components/MessageError/index.js

```diff
@@ -0,0 +1,44 @@
+import { useRouter } from "next/router";
+import styled from "styled-components";
+import SVGIcon from "../Icons";
+
+export default function MessageError() {
+  const router = useRouter();
+
+  return (
+    <MessageContainer>
+      <MessageHeadline>Fehler beim Laden</MessageHeadline>
+      <MessageButton onClick={router.reload}>
+        <SVGIcon variant="refresh" width="80px" />
+        <MessageText>Neu laden</MessageText>
+      </MessageButton>
+    </MessageContainer>
```

Comment:
As the name of your component is already `MessageError` it is alright to not use "Message" again for each styled component.

Reply:
fixed